etc/flow/global.cylc

```diff
@@ -0,0 +1,13 @@
+[platforms]
+    [[.*cross]]
+        [[[meta]]]
+            title = "Kings/Charing/Bounds Cross"
+            description = """
+                If platform name is a regex you might want
+                an explicit title
+            """
+    [[mornington_crescent]]
+        [[[meta]]]
+            description = """
+                I win!
+            """
```

Comment:
These files are missing terminating newlines.

You can configure your text editor not to do this.

Reply:
In VSCode `"files.insertFinalNewline": true`

Reply:
This is a localized problem - I'm sure you'd have noticed before now if it were my editor settings!